### DISCUSSION TITLE
求大佬解答

### DISCUSSION THREAD

**Discussion by ironjing** [teslamate-org/teslamate#2620] [Q&A]
这个数据，是从车机上传到服务器的？还是从特斯拉官方拉取的？不断上传，耗电吗？拉取频率怎么样？谢谢

**Comment by ktsenvolt** [comment]
https://www.youtube.com/watch?v=t34z3tHz42U&ab_channel=FrederikBov%C3%A9
![image](https://user-images.githubusercontent.com/80360674/165652959-e6d92622-9df7-4541-84b6-58d63929e433.png)

**Reply by ironjing** [reply]
Thank you!

**Comment by 0x11-dev** [comment]
从体感上，tesla app 在一段时间不使用车机上报会降低频率，打开tesla后需要等待一段时间才能获取新的数据。 猜测通过某种信令下行唤醒(可能是GSM短信)。 所以频繁打开tesla理论上增加耗电
